contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -840,160 +840,4 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2, PricesV1 {
-        StakingPoolMigrationData(
-          HUGH, // stakerAddress
-          HUGH, // managerAddress
-          'QmP4qU9o4WqT2TLHzieFJQpQfECy5bTcyRjikjRPs57UNa', // ipfsDescriptionHash
-          false, // isPrivatePool
-          10, // initialPoolFee
-          20, // maxPoolFee
-          deposit,
-          trancheStakeRatio // stake on each tranche, as % out of the deposit
-        ),
-        products
-      );
-    } else if (stakerAddress == ARMOR_STAKER) {
-
-      uint armorAAALowRiskPoolDeposit = deposit * 75 / 100; // 75% of the deposit
-      uint[8] memory trancheStakeRatioAAA = [uint(20), 25, 25, 15, 10, 0, 0, 0];
-
-      expectedMigratedAmount += createPoolAndDeposit(
-        StakingPoolMigrationData(
-          ARMOR_STAKER, // stakerAddress
-          ARMOR_MANAGER, // managerAddress
-          'QmQDwQDLDnvHn4ajodwGNiHGYwqBrZvy2Y9kMAvvtfWF45', // ipfsDescriptionHash
-          false, // isPrivatePool
-          15, // initialPoolFee
-          25, // maxPoolFee
-          armorAAALowRiskPoolDeposit, // deposit
-          trancheStakeRatioAAA // stake on each tranche, as % out of the deposit
-        ),
-        products
-      );
-
-      uint armorAAMidRiskPoolDeposit = deposit - armorAAALowRiskPoolDeposit; // 25% of the deposit
-      uint[8] memory trancheStakeRatioAA = [uint(20), 25, 25, 15, 10, 0, 0, 0];
-
-      expectedMigratedAmount += createPoolAndDeposit(
-        StakingPoolMigrationData(
-          ARMOR_STAKER, // stakerAddress
-          ARMOR_MANAGER, // managerAddress
-          'QmS76eiGiC9KxrYQJ3WByvfJzD28CzELpjSd4V4ywmwNQP', // ipfsDescriptionHash
-          false, // isPrivatePool
-          15, // initialPoolFee
-          25, // maxPoolFee
-          armorAAMidRiskPoolDeposit,
-          trancheStakeRatioAA // stake on each tranche, as % out of the deposit
-        ),
-        products
-      );
-    } else if (stakerAddress == NM_FOUNDATION) {
-
-      uint[8] memory trancheStakeRatio = [uint(0), 25, 0, 25, 0, 0, 0, 0];
-
-      expectedMigratedAmount += createPoolAndDeposit(
-        StakingPoolMigrationData(
-          NM_FOUNDATION, // stakerAddress
-          NM_FOUNDATION, // managerAddress
-          'QmWXrQNe7gdwo42s4grJVDG9q92Z2Eu9FsvWUPgoYUyVzs', // ipfsDescriptionHash
-          true, // isPrivatePool
-          0, // initialPoolFee
-          99, // maxPoolFee
-          deposit,
-          trancheStakeRatio // stake on each tranche, as % out of the deposit
-        ),
-        products
-      );
-    }
-
-    uint nxmBalanceAfter = nxm.balanceOf(address(this));
-    uint actualMigratedAmount = nxmBalanceBefore - nxmBalanceAfter;
-    require(actualMigratedAmount == expectedMigratedAmount, "NXM balance mismatch");
-
-    // Send unlocked NXM back
-    uint nxmToBeUnlocked = deposit - actualMigratedAmount;
-    if (nxmToBeUnlocked != 0) {
-      nxm.transfer(stakerAddress, nxmToBeUnlocked);
-    }
-  }
-
-  function migrateToExistingV2Pool(uint stakingPoolId, uint trancheId) external {
```

Comment:
Please make sure the frontend is not using this function

Reply:
Checked - it's not used by frontend